packages/booking/package.json

```diff
@@ -41,11 +44,12 @@
     "babel-loader": "^9.1.0",
     "eslint": "^8.19.0",
     "jest": "^29.2.1",
+    "metro-react-native-babel-preset": "^0.73.9",
     "prettier": "^2.8.8",
-    "metro-react-native-babel-preset": "0.73.9",
-    "react-test-renderer": "18.2.0",
+    "react-test-renderer": "^18.2.0",
+    "super-app-showcase-sdk": "0.0.1",
```

Comment:
let's make sure adding `super-app-showcase-sdk` is the only dep changed in this PR